src/components/ContactPage/ContactPage.js

```diff
@@ -108,12 +108,12 @@ const ContactPage = () => {
                 <Row className="mt-5 mb-5">
                     <Col md={5} className="text-center">
                         <FaPhoneAlt className="contact-icons" />
-                        <h5 className="contact-info mt-3"> (347)-997-0315</h5>
+                        <h5 className="contact-info mt-3"><a href="tel:3479970315"> (347)-997-0315</a></h5>
```

Comment:
Client requested there should be a clickable link for user to call the number and email as well for the contact page